### ISSUE TITLE
Conflict with Composer

### ISSUE THREAD

**Issue by D3nnis3n** [oetterer/BootstrapComponents#48]
Hey,
I'm using the following composer.local.json file in the root:

```
{
	"extra": {
		"merge-plugin": {
			"include": [
				"extensions/*/composer.json",
				"skins/*/composer.json"
			]
		}
	},
	"require": {
		"mediawiki/bootstrap": "~5.0",
		"mediawiki/bootstrap-components": "~5.2",
		"mediawiki/chameleon-skin": "~5.0@dev"
	}
}
```

Unfortunately composer can't deal with that:
![image](https://github.com/user-attachments/assets/e216d4d0-f85a-4ada-84b9-d26ffdc8a820)

Is that something you could address? It's the only plugin causing an issue for some reason.

(Tried to switch back to composer after 5.2 was released from the manually installed master branch)

**Comment by oetterer** [comment]
@D3nnis3n could you please try  againg with 5.2.1?

**Comment by D3nnis3n** [comment]
Works like a charm, thanks so much :)

**Comment by oetterer** [comment]
closed by bec7dd3e864d61df620008143ebe879de70d5f0f
